### PR TITLE
Fix: Number of files in delta reporter

### PR DESCRIPTION
**Pull request by ulasozguler**
When number of files with coverage change is less then total number of files in coverage report, `DeltaReporter.per_file_stats` throws a "IndexError: list index out of range" exception. 

In test case, both coverage files have 3 items. One of the items in `new-coverage.xml` has coverage change.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- Fix `pycobertura diff` bug related to number of changed files. Thanks @ulasozguler
```
Comment (aconrad) [inline CHANGES.md:5]: Can we be more specific? Maybe reuse the wording used in the PR's description.

Comment (ulasozguler) [reply CHANGES.md]: Done.

**Review by aconrad — APPROVED** [review]
Thanks for fixing this! Much appreciated!

**Comment by aconrad** [comment]
Released as v3.2.2

**Pull request merged**